feature-detects/pointerevents.js

```diff
@@ -2,6 +2,7 @@
 {
   "name": "DOM Pointer Events API",
   "property": "pointerevents",
+  "caniuse": "pointer-events",
```

Comment:
no, this caniuse tag refers to the CSS pointer-events not the DOM pointer-events (two different things and tests)

Reply:
Okay I think that the one we are looking for is [pointer](https://caniuse.com/#feat=pointer)